spec/controllers/application_controller_spec.rb

```diff
@@ -86,10 +86,36 @@ def index
         alice.update_attribute(:getting_started, true)
       end
 
-      it "redirects to getting started if the user has getting started set to true" do
+      it "redirects to getting started if the user has getting started set to true and a blank profile" do
         expect(@controller.send(:after_sign_in_path_for, alice)).to eq(getting_started_path)
       end
     end
+
+    context 'getting started true and one tag present on user' do
+      before do
+        alice.update_attribute(:getting_started, true)
+        @tag = ActsAsTaggableOn::Tag.create!(:name => "partytimeexcellent")
+        allow(@controller).to receive(:current_user).and_return(alice)
+        TagFollowing.create!(:tag => @tag, :user => alice )
+      end
+
+      it "redirects to stream if the user has getting started set to true and has already added tags" do
+        expect(@controller.send(:after_sign_in_path_for, alice)).to eq(stream_path)
+      end
+    end
+
+    context 'getting started true and user image present on user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.